src/logic/utils.ts

```diff
@@ -1,14 +1,28 @@
 import {
     NEW_PLAYER_HIGHLIGHT_DAYS,
+    NUMBER_OF_PLAYERS_FOR_VALID_MATCH,
+    PLAYER_MAXIMUM_GAMES_AS_NEW_PLAYER,
     PLAYER_MINIMUM_GAMES_REQUIRED,
     PLAYER_MINIMUM_WINS_REQUIRED
 } from "../components/constants";
 import { Player } from "../types/domain/Player";
 
+
+/**
+ * Gets the win rate as a percentage (number, without the "%").
```

Comment:
the message here is good, but can we rephrease his to :
Gets the win rate as a percentage as a number from 0 to 100. 

you don't need to mention that the percentage is not included, since numbers cannot have percentage symbol in them.

Reply:
fiiiine. fixed.